backend/src/routes/v2/workspace.ts

```diff
@@ -118,4 +118,19 @@ router.delete( // TODO - rewire dashboard to this route
 	workspaceController.deleteWorkspaceMembership
 );
 
+
+router.post(
+	'/:workspaceId/settings',
```

Comment:
Can you change this endpoint to PATCH `/:workspaceId/auto-capitalization` instead to be more descriptive and reflect that it's an update to a resource as opposed to creating a new resource?